src/utils.ts

```diff
@@ -65,30 +65,27 @@ export function makeTmpDir() {
       return makeUnixTmpDir();
     case 'wsl':
       // We populate the user's Windows temp dir so the folder is correctly created later
-      process.env.TEMP = getLocalAppDataPath(`${process.env.PATH}`);
+      process.env.TEMP = getWSLLocalAppDataPath(`${process.env.PATH}`);
     case 'win32':
       return makeWin32TmpDir();
     default:
       throw new UnsupportedPlatformError();
   }
 }
 
-export function toWinDirFormat(dir: string = ''): string {
-  const results = /\/mnt\/([a-z])\//.exec(dir);
-  if (!results) {
-    return dir;
-  }
+export function toWin32Path(dir: string = ''): string {
+  return execFileSync('wslpath', ['-w', dir]).toString().trim();
```

Comment:
this works even if the dir is already a win32 path, right? before we had that regex noop business to guard against it

Reply:
It won't error, but it will give you an invalid path. Unfortunately there's not a nice single line solution, but the noop is probably the best way to go; you just need to make sure that you're always using absolute paths.

Reply:
(resolving since I added a noop test)